articles/azure-monitor/insights/container-insights-region-mapping.md

```diff
@@ -61,6 +61,8 @@ The AKS cluster resources or Log Analytics workspace can reside in other regions
 |WestUS |WestUS |
 |WestUS2 |WestUS2 |
 |WestCentralUS<sup>1</sup>|EastUS<sup>1</sup>|
+|**US Government(Fairfax)** | |
+|USGov Virginia |USGov Virginia|
```

Comment:
Line 65 should be rewritten to - |US Government Virginia |
Line 64 should be removed because US Government Virginia is "Fairfax"

Reply:
updated, does this look right now?